doc/developer/server_side.rst

```diff
@@ -53,6 +53,17 @@ of an existing c2cgeoportal application. Here's how:
 
     rm .build/requirements.timestamp && make -f <user>.mk build
 
+.. note:: Print performance issue
+
+   When we restart the print frequantly we can have performance issue
```

Comment:
When restarting the print server frequently, performance issues may randomly be observed.